MAUI/DataGrid/performance.md

```diff
@@ -36,4 +36,49 @@ N> Download demo application from [GitHub](https://github.com/SyncfusionExamples
 * Both implicit and explicit padding are not supported. However, left and right padding will be applied based on the padding, while the top and bottom positions will be adjusted based on the content.
 * [DataGridColumn.LineBreakMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_LineBreakMode) is not supported.
 * Runtime theme changes will not be applied.
-* This is not supported for [DataGridTemplateColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridTemplateColumn.html), [DataGridCheckBoxColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCheckBoxColumn.html) and [DataGridImageColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridImageColumn.html).
\ No newline at end of file
+* This is not supported for [DataGridTemplateColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridTemplateColumn.html), [DataGridCheckBoxColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCheckBoxColumn.html) and [DataGridImageColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridImageColumn.html).
+
+
+# Incremental loading
+
+DataGrid supports to load the data incrementally using `ISupportIncrementalLoading` interface. `ISupportIncrementalLoading` interface has `LoadMoreItemsAsync` method which helps to load the data incrementally. `LoadMoreItemsAsync` called in on-demand while scrolling based on `HasMoreItems` property.
+
+If `HasMoreItems` is false, SfDataGrid stops calling `LoadMoreItemsAsync`. SfDataGrid have `IncrementalList` which is derived from `ISupportIncrementalLoading`. You can use IncrementalList or create collection derived from `ISupportIncrementalLoading` and bind it `SfDataGrid.ItemsSource`.
+
+Additionally it supports `SfDataGrid.DataFetchSize` , which will automatically fetch the data when the user reaches the end of the  `dataGrid`. 
+
+In the below code, `IncrementalList` is initialized by passing Action to its constructor for loading items incrementally.
```

Comment:
The DataGrid supports loading data incrementally using the `ISupportIncrementalLoading` interface. This interface includes the `LoadMoreItemsAsync` method, which helps to load data incrementally. The `LoadMoreItemsAsync` method is called on-demand while scrolling, based on the `HasMoreItems` property.

If `HasMoreItems` is false, SfDataGrid stops calling `LoadMoreItemsAsync`. SfDataGrid includes `IncrementalList`, which is derived from `ISupportIncrementalLoading`. You can use `IncrementalList` or create a collection derived from `ISupportIncrementalLoading` and bind it to `SfDataGrid.ItemsSource`.

Additionally, the `SfDataGrid.DataFetchSize` property defines the count argument in the `LoadMoreItemsAsync` method, which determines the amount of data to be loaded into the items source when the user reaches the end of the `SfDataGrid`

In the code below, `IncrementalList` is initialized by passing an Action to its constructor for loading items incrementally.